granulate_utils/linux/cgroups_v2/cgroup.py

```diff
@@ -98,7 +98,7 @@ def _find_v1_hierarchies() -> Mapping[str, tuple[str, str]]:
     return hierarchies
 
 
-def _find_v2_hierarchy() -> Optional[tuple[str, str]]:
+def find_v2_hierarchy() -> Optional[tuple[str, str]]:
```

Comment:
Why are we making this change as part of this PR?

Reply:
Ah - because they were exported functions by the old `.linux.cgroups` module,. Got it.